src/video.js

```diff
@@ -64,3 +66,47 @@ export const checkVideoBidSetup = hook('sync', function(bid, adUnit, videoMediaT
 
   return true;
 }, 'checkVideoBidSetup');
+
+export function registerVASTTrackers(tracker) {
```

Comment:
We just discussed how this should be treated in terms of privacy (GDPR & co), and concluded that adding trackers should be guarded by the `reportAnalytics` activity. In practice this means that:

- registerVASTTrackers needs to ask for who's tracking, similar to `storageManager` - for example `registerVASTTrackers(moduleType, moduleName, trackerFn)` 
- getVASTTrackers should check whether `reportAnalytics` is allowed before invoking the generator function - e.g. 
    ```javascript
       import {isActivityAllowed} from './activities/rules.js';
       import {activityParams} from './activities/activityParams.js';
       import {ACTIVITY_REPORT_ANALYTICS} from './activities/activities.js';
    
       // ...
       vastTrackers
          .filter(
              ({moduleType, moduleName, trackerFn}) => isActivityAllowed(ACTIVITY_REPORT_ANALYTICS, activityParams(moduleType, moduleName))
           ).forEach(({trackerFn}) => {
              // ... run fn and collect trackers
           })
    ```

Reply:
It isn't, and for analytics modules it makes no difference because they already need `reportAnalytics`, but if this is done from an RTD module we need to make sure that it's blocked when there's no consent for the tracking.

Reply:
I've juste made the suggested change, can you confirm this is what you were talking about ?

Reply:
Confirmed!